tests/test_precovery_index.py

```diff
@@ -90,16 +91,20 @@ def test_precovery(test_db_dir):
 
     # For each sample orbit, validate we get all the observations we planted
     for orbit in orbits_keplerian:
-        results = precover(orbit, test_db_dir, tolerance=1 / 3600, window_size=1)
+        matches, misses = precover(
+            orbit, test_db_dir, tolerance=1 / 3600, window_size=1
+        )
```

Comment:
I like this language - "matches and misses" - a lot more than "PrecoveryCandidates and FrameCandidates." Maybe we can switch to that?

Reply:
I was thinking about that too. If the rest of the team is okay with these high level changes I'd be happy to use that new language. In particular, FrameCandidate had me confused for a long time initially.

Reply:
I like this too. Though I'm wondering if we can maintain some clarity on the fact that these really aren't "matches" without additional validation (OD, cutouts, etc..). I'll defer to the two of you to decide if you think that matters.

Reply:
That's an interesting point. We don't want anyone naively assuming that if precovery returns a result, it is a match for their object. However, it's also true that what is being returned is a match for the parameters sent in.